assets/src/stories-editor/constants.js

```diff
@@ -106,7 +106,7 @@ export const ALLOWED_BLOCKS = [
 
 export const IMAGE_BACKGROUND_TYPE = 'image';
 export const VIDEO_BACKGROUND_TYPE = 'video';
-export const ALLOWED_MEDIA_TYPES = [ 'image', 'video/mp4' ];
+export const ALLOWED_MEDIA_TYPES = [ 'image', 'video' ];
```

Comment:
This constant is only used for the Background Media and its poster image.

`video/mp4` didn't seem to prevent other video mime types, like `video/quicktime`.

I think these are intended to be types, like `attachment.get( 'type' )` would return. The only time I saw `video/mp4` was with `attachment.get( 'mime' )`

Reply:
The new function `isFileTypeAllowed` ensures that videos have the right mime type.